app/assets/javascripts/application.js

```diff
@@ -11,17 +11,11 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
 //= require jquery_ujs
-//= require jquery.turbolinks
-//= require turbolinks
-//= require turbolinks-compatibility
-//= require jquery-ui/core
+//= require datatables
+//  require turbolinks
```

Comment:
Happy to remove it.  I had left it in hopes that we could figure out an approach to getting it working.